app/qml/form/editors/MMFormCalendarEditor.qml

```diff
@@ -58,7 +59,7 @@ MMPrivateComponents.MMBaseSingleLineInput {
   hasCheckbox: _fieldRememberValueSupported
   checkboxChecked: _fieldRememberValueState
 
-  readOnly: _fieldIsReadOnly
+  readOnly: _fieldFormIsReadOnly
```

Comment:
Sorry @VitorVieiraZ, I had a second thought about it and I believe we need to approach it from a slightly different perspective. 
Our current solution would keep fields with "Apply default value on update" in editable state. This would show icons / actions of the field as if one could click on them, see 
<img width="311" alt="image" src="https://github.com/MerginMaps/mobile/assets/22449698/3147751c-e148-4603-86a6-8f86b7451de1">

But in reality, they should be disabled.

We need to keep such fields disabled though.
<img width="312" alt="image" src="https://github.com/MerginMaps/mobile/assets/22449698/2aa23c07-5e84-4352-bc00-49d9665bd422">

---

Thus, I suggest to do the following:
 - add `property bool shouldShowValidationMsg` to `MMBaseInput` and use it as a condition here https://github.com/MerginMaps/mobile/blob/1c823dc8c732936392fda0f17b7d8ea056215716/app/qml/components/private/MMBaseInput.qml#L173 instead of the `root.editState === "enabled"`
 - then set the `shouldShowValidationMsg` in every form editor to `!_fieldFormIsReadOnly`
 - then revert the value of the `readOnly` in all editors to `readOnly: _fieldFormIsReadOnly || !_fieldIsEditable`
 - with this, we do not need to change anything in `MMBaseSingleLineInput.qml` and validation messages will be visible for all fields, no matter if they are editable or not.

This solution is more flexible, does not change already working workflows and should not bring any unwanted changes 🤞🏻.